mappings/net/minecraft/client/render/chunk/ChunkBuilder.mapping

```diff
@@ -47,12 +64,15 @@ CLASS net/minecraft/unmapped/C_cifoaixd net/minecraft/client/render/chunk/ChunkB
 		FIELD f_srtroeov blockEntities Ljava/util/Set;
 		FIELD f_tgbwoiht index I
 		FIELD f_tiqhepqc sortTask Lnet/minecraft/unmapped/C_cifoaixd$C_lcybgaxw$C_temnlrwc;
+		FIELD f_txnojlbs CHUNK_SIZE I
```

Comment:
Just `SIZE`, chunk size is redundant inside `BuiltChunk`

Reply:
```suggestion
		FIELD f_txnojlbs SIZE I
```